syftbox/tui/widgets/sync_widget.py

```diff
@@ -0,0 +1,65 @@
+from typing import Any, List, Optional
+
+import requests
+from textual.binding import Binding
+from textual.containers import ScrollableContainer
+from textual.widgets import DataTable, Input, Label, Static
+
+from syftbox.client.plugins.sync.local_state import SyncStatusInfo
+from syftbox.lib import Client
+
+
+class SyncWidget(Static):
+    LIMIT = 100
+    BINDINGS = [
+        Binding("f", "focus_search", "Search", show=True),
+    ]
+
+    def __init__(self, syftbox_context: Client) -> None:
+        super().__init__()
+        self.syftbox_context = syftbox_context
+        self.default_filter = f"{syftbox_context.email}/**"
+
+    def compose(self) -> Any:
+        yield Label("Filter Files")
+        yield Input(value=self.default_filter, placeholder="glob pattern", id="path_filter")
```

Comment:
It wasn't clear to me that there is an `Input` field here. Can we make it easier to distinguish? Maybe brighter bg color and a short description would be nice. Something like: 'Enter a glob pattern to filter files:'